src/functionalTest/resources/features/F-010/S-107.td.json

```diff
@@ -0,0 +1,76 @@
+{
+  "_guid_": "S-107",
+  "title": "must successfully create judicial role mapping for Employment Judge - Fee Paid appointment",
+  "_extends_": "F-010_Test_Data_Base",
+  "specs": [
+    "contains the actorIds of the user just published"
+  ],
+  "method": "POST",
+  "uri": "{{ROLE_ASSIGNMENT_URL}}/am/role-assignments/query",
+  "request": {
+    "headers": {
+      "sort": "roleName",
+      "direction": "ASC",
+      "Content-Type": "application/vnd.uk.gov.hmcts.role-assignment-service.post-assignment-query-request+json;charset=UTF-8;version=2.0"
+    },
+    "body": {
+      "queryRequests":[
+        {
+          "actorId": "${[scenarioContext][childContexts][S-107_DeleteDataForRoleAssignments][testData][request][body][userIds]}",
+          "attributes": {
+            "jurisdiction": ["EMPLOYMENT"]
+          }
+        }
+      ]
+    }
+  },
+  "expectedResponse": {
+    "_extends_": "Common_200_Response",
+    "body": {
+      "roleAssignmentResponse": [
+        {
+          "id": "[[ANYTHING_PRESENT]]",
+          "actorIdType": "IDAM",
+          "actorId": "${[scenarioContext][testData][request][body][queryRequests][1][actorId][0]}",
```

Comment:
The pipeline index errors may be due to the `[queryRequests][1]` when there is only 1 entry in the array, i.e. it should be `[queryRequests][0]` or you are missing an entry in the request body.


NB: S-106 does have two entries in the queryRequests array however both S-107 & S-108 have only 1.

Reply:
Resolved in b91f4f362a6268b16a99a9506d33adbb9aae43e9.